main.go

```diff
@@ -167,6 +172,8 @@ func main() {
 			setupLog.Error(err, "unable to create controller", "controller", "SecretSync")
 			os.Exit(1)
 		}
+		// Part of initial configuration check, but only done on the control-plane
+		secret.VerifyReplicatedSecret(ctx, uncachedClient, options.LeaderElectionID, watchNamespace, contextNames)
```

Comment:
Shouldn't we handle the returned error here?

Reply:
I'm moving this to per-cluster-replicatedSecret.